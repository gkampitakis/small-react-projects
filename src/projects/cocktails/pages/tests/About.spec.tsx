export {};

describe('About', () => {
  it.todo('Should render correctly');
});
