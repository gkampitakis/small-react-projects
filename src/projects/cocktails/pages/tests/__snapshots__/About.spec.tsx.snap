// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Should render correctly 1`] = `
<section
  className="about-section"
>
  <h1
    className="title"
  >
    about us
  </h1>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Molestiae repudiandae architecto qui adipisci in officiis, aperiam sequi atque perferendis eos, autem maiores nisi saepe quisquam hic odio consectetur nobis veritatis quasi explicabo obcaecati doloremque? Placeat ratione hic aspernatur error blanditiis?
  </p>
</section>
`;
